### ISSUE TITLE
Embed within scijava-ops

### ISSUE THREAD

**Issue by tinevez** [clEsperanto/clesperantoj_prototype#31]
There is a new version of Ops by the Eliceiri/LOCI lab: 
https://forum.image.sc/t/announcing-scijava-ops-v1-0-0/97235

The Ops library aims at being an extensible collection of image processing algorithms. It is built so that it can wrap or include 3rd party libraries within a centralized syntax, and offer a 'matching' mechanism that finds the best algorithm given the user inputs. The goal and philosophy is summarized in the wiki for the old version of ops: https://imagej.net/libs/imagej-ops/

Could clEsperantoJ be a backend for ops? It would grant ops with GPU accelerated processing, and give clEsperanto more reach.

**Comment by haesleinhuepf** [comment]
Yes, that would technically work! We had a clij-ops integration, which was mostly generated code btw. It was abandoned (and thus, there was no clij2-ops integration) because nobody was using it and nobody had the time to maintain it.

https://github.com/clij/clij-ops

**Comment by tinevez** [comment]
Let's discuss when we have a first fully working IJ version?
